backend/src/StamAcasa.Common/DTO/UserInfo.cs

```diff
@@ -17,7 +17,7 @@ public class UserInfo {
         public string City { get; set; }
         public int Age { get; set; }
         public Gender Gender { get; set; }
-        public string PreexistingMedicalCondition { get; set; }
+        public string[] PreexistingMedicalCondition { get; set; }
```

Comment:
Maybe renaming to `PreexistingMedicalConditions` to show the fact is a list now? Small thing, we can do it later also